hydra/test_utils/launcher_common_tests.py

```diff
@@ -346,6 +348,40 @@ def sweep_two_config_groups(
 
 @pytest.mark.usefixtures("hydra_restore_singletons")
 class IntegrationTestSuite:
+    working_dir_key = "tests_working_dir"
+    output_dir_key = "tests_output_dir"
+
+    def filter_overrides(
```

Comment:
You are adding a whole lot of new logic here and I have no idea what it's supposed to do.
1. What overrides is this filtering?
2. What does this achieve?

Reply:
I want to override the tests working dir and tests output dir somehow. 

Right now I'm passing the override from here https://github.com/facebookresearch/hydra/pull/518/files#diff-b44e4fd91c6f6ff3f7dc895d9a6a1672R121-R122

So in the tests I want to get the dir overrides value but also remove them from the application overrides. Not sure if there's a better way to do this (previous I was doing method overriding). Any suggestion?

Reply:
Hydra provices the ability to override EVERYTHING in the configuration through the command line overrides.
you can pass an override like : hydra.run.dir=/foo/bar and that will be come the run dir for the job, no?

Reply:
Thx. I've taken another pass and try to override everything instead for the Integration tests.